src/components/drawer/drawer.js

```diff
@@ -0,0 +1,39 @@
+import angular from "angular";
+import ngAnimate from "angular-animate";
+import ngAria from "angular-aria";
+import ngMaterial from "angular-material";
+import template from "./drawer.html";
+import { accountsModule } from "../accounts/accounts";
+import { tagsModule } from "../tags/tags";
+
+export class DrawerController {
+  constructor($mdSidenav) {
+    this.$mdSidenav = $mdSidenav;
+    this.isOpened = false;
+  }
+  handleAccountClick(event) {
+    console.log("handleAccountClick");
+  }
+
+  addTagClick(event) {
+    console.log("addTagClick");
+  }
+
+  $onChanges(changes) {
+    this.toggleLeft();
```

Comment:
you do not need this -- you can directly use `isDrawerOpened ` binding in template